### PR TITLE
Add Schedule

### PR DESCRIPTION
**Pull request by robertdijk**
This PR adds a way to make a nice schedule on the website, where each activity has a page.

### CODE REVIEW

**Comment by robertdijk** [comment]
For now, I have the schedule page as a draft.
The items and their times in the schedule are not final, and can easily be changed :).
A limitation of this schedule is that it only works per half an hour.

**Comment by mpsijm** [comment]
Oohh, looking fancy! :star_struck: It looks like the schedule is slightly outdated by now, though :stuck_out_tongue: And I assume that most of the pages will get some more content later as well? :slightly_smiling_face:

**Comment by robertdijk** [comment]
Update, the schedule now works for every 5 minutes, instead of 30 minutes

**Comment by mpsijm** [comment]
Is it just me, or is the font size really small? :thinking: 

image

**Comment by mpsijm** [comment]
The content of the schedule is looking good (and consistent with the schedule on Drive) :smile: Two more comments:

- Would it be nice to show a legend somewhere that shows which colour means what? Or to show in the blocks who are expected to be where (participants, coaches, or both)?
- I see a "Seminar" in the schedule on drive, is that not part of the public schedule?

File: content/schedule/coach_meeting_2.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "General coach meeting"
+type: event
+day: sunday
+start: 13:30
+duration: 1h30m
+width: 1
+location: Aula - Lecture Hall B
```
Comment (verwoerd) [inline content/schedule/coach_meeting_2.md:8]: Didn't we move this to the Auditorium?

File: content/schedule/late_registration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+type: event
+day: saturday
+start: 8:30
+duration: 1h0m
+width: 2
+location: Aula
+people: all
+---
+
+Here should be some information about who should be at the registation.
```
Comment (verwoerd) [inline content/schedule/late_registration.md:12]:
```suggestion
During registration all team members and on-site coaches need to register. You will receive your badge, T-Shirt and goody bag. Also a team picture will be taken.
```

File: content/schedule/lunch.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+day: saturday
+start: 12:00
+duration: 1h30m
+width: 1
+location: Aula - Canteen
+people: all
+---
+
+## Location
+The speech takes place in the Canteen of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
```
Comment (verwoerd) [inline content/schedule/lunch.md:13]:
```suggestion
The lunch takes place in the Canteen of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
```

File: content/schedule/lunch.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Lunch"
```
Comment (verwoerd) [inline content/schedule/lunch.md:2]:
```suggestion
title: "Lunch and Company Market"
```

File: content/schedule/registration.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+type: event
+day: friday
+start: 14:00
+duration: 8h0m
+width: 1
+location: Aula
+people: all
+---
+
+Here should be some information about who should be at the registation.
```
Comment (verwoerd) [inline content/schedule/registration.md:12]:
```suggestion
During registration all team members and on-site coaches need to register. You will receive your badge, T-Shirt and goody bag. Also a team picture will be taken.

```

**Review by verwoerd — APPROVED** [review]
Some minor improvements/suggestions. Further looks great.

**Comment by robertdijk** [comment]
> Is it just me, or is the font size really small? 🤔
> 
> image

Yes, but otherwise it doesn't fit on mobile. I think its not too small.

**Comment by robertdijk** [comment]
> The content of the schedule is looking good (and consistent with the schedule on Drive) 😄 Two more comments:
> 
> * Would it be nice to show a legend somewhere that shows which colour means what? Or to show in the blocks who are expected to be where (participants, coaches, or both)?
I think it speaks for itself.

> * I see a "Seminar" in the schedule on drive, is that not part of the public schedule?
I added it!

**Pull request merged**